src/components/image-slider/ImageSlider.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import "./ImageSlider.css";
+
+const ImageSlider = ({ images, parent }) => {
+  // takes in images as props
+  const [index, setIndex] = useState(0); // create state to keep track of images index, set the default index to 0
+
+  const slideRight = () => {
+    setIndex((index + 1) % images.length); // increases index by 1
+  };
+
+  const slideLeft = () => {
+    const nextIndex = index - 1;
+    if (nextIndex < 0) {
+      setIndex(images.length - 1); // returns last index of images array if index is less than 0
+    } else {
+      setIndex(nextIndex);
+    }
```

Comment:
Can we write this using ternary? 
```suggestion
    nextIndex<0 ? setIndex(images.length - 1) : setIndex(nextIndex)
```